### ISSUE TITLE
Add Support for STSAssumeRoleSessionCredentialsProvider

### ISSUE THREAD

**Issue by famuyiwad** [logstash-plugins/logstash-input-kinesis#34]
Could you add a support for `STSAssumeRoleSessionCredentialsProvider` so that credentials can be retrieved via AWS Security Token Service (STS) using a configuration like this.

```
[sts]
role_arn = arn:aws:iam::123456:role/roleSessionName
source_profile = default
```

Thanks in advance

**Comment by codekitchen** [comment]
I don't believe anybody has decided to work on it, somebody will need to submit a PR for the enhancement.

**Comment by robbavey** [comment]
I believe this has been addressed by #57, which is available in version `2.0.11` of the plugin
